com.unity.formats.fbx/Editor/Sources/Recorders/FbxRecorder/FbxRecorder.cs

```diff
@@ -39,7 +39,9 @@ protected override void EndRecording(RecordingSession session)
                 var clipName = absolutePath.Replace(FileNameGenerator.SanitizePath(Application.dataPath), "Assets");
                 
 #if UNITY_2018_3_OR_NEWER
-                aInput.GameObjectRecorder.SaveToClip(clip, settings.FrameRate);
+                var options = new Animations.CurveFilterOptions();
+                options.keyframeReduction = false;
+                aInput.GameObjectRecorder.SaveToClip(clip, settings.FrameRate, options);
```

Comment:
Given 2018.3 is the oldest we support we don't need the #if. Unless that #if is supposed to read 2019.3